core/bentley/src/Logger.ts

```diff
@@ -58,6 +117,8 @@ export class Logger {
   protected static _logTrace: LogFunction | undefined;
   private static _categoryFilter: Map<string, LogLevel> = new Map<string, LogLevel>();
   private static _minLevel: LogLevel | undefined = undefined;
+  private static _tracer?: Tracer;
+  private static _openTelemetry?: { trace: TraceAPI, context: ContextAPI };
```

Comment:
sanity check: is this alright, or do I need to have `@opentelemetry/api` at least in peerDependencies for this to work reliably? I haven't noticed any issues when testing it with GPB, but I don't know if there are any edge cases.

Reply:
I _think_ that should be almost completely safe, but the fact that `ContextAPI` is a class type w/ a private member concerns me...  I'm pretty sure without peerDependencies you could hit issues w/ mis-matched versions...

Would this still work if you changed the type to:

```ts
private static _openTelemetry?: { trace: TraceAPI, context: Pick<ContextAPI, "active" | "with"> };
```

I think that would probably be safest...

Reply:
That's what I was worried about. Thanks for reminding me about `Pick`, I was thinking about how I would selectively pick the definitions of specific methods :)